lib/statesman/adapters/active_record_queries.rb

```diff
@@ -105,6 +111,14 @@ def states_where(temporary_table_name, states)
           end
         end
 
+        def most_recent_transition_alias
+          "most_recent_#{transition_name}"
```

Comment:
Brevity is nice, but I went for `most_recent` for consistency with the column name. Not particularly attached though.